self.js

```diff
@@ -72,6 +72,12 @@ self.on('ready', () => {
       self.editStatus(config.defaultStatus.toLowerCase(), {name: games[~~(Math.random() * games.length)]})
     }, config.rotatePlayingGameTime) // Edits playing game every X milliseconds (You can edit this number in the config file)
   }
+  if (config.rotateStreamingGame && streams.length > 0) {
+    log.log('Changing streaming status every ' + (config.rotateStreamingStatusTime / 1000) / 60 + ' minutes.', 'Config')
+    setInterval(() => {
+      self.editStatus(config.defaultStatus.toLowerCase(), {name: games[~~(Math.random() * games.length)], type: 1, url: 'https://www.twitch.tv/twitch'})
+    }, config.rotateStreamingStatusTime) // Edits streaming status every X milliseconds (You can edit this number in the config file)
+  }
```

Comment:
`games[]` is actually referring to the `games.json` file and not the `streams.json` one, you should require it at the top of the file and then change `games[]` to `streams[]`